client/blocks/login/social.jsx

```diff
@@ -153,21 +153,37 @@ class SocialLoginForm extends Component {
 			redirectTo &&
 			redirectTo.includes( 'jetpack/connect' ) &&
 			config.isEnabled( 'jetpack/magic-link-signup' );
+
+		const privacyLink = (
+			<a
+				href={ localizeUrl( 'https://automattic.com/privacy/' ) }
+				target="_blank"
+				rel="noopener noreferrer"
+			/>
+		);
+		const tosLink = (
+			<a
+				href={ localizeUrl( 'https://wordpress.com/tos/' ) }
+				target="_blank"
+				rel="noopener noreferrer"
+			/>
+		);
```

Comment:
We seem to do things in the order `tosLink`, `privacyLink` elsewhere, so this upsets my Obsessive Compulsive Disorder, but I'll live.

Reply:
@andfinally , I changed the order in https://github.com/Automattic/wp-calypso/pull/66220/commits/55dc9c022dec8a3ff001ef4d9504508c8bd229bb so you won't have to worry about it in the future 😉